### PR TITLE
Audit logs for comment entity

### PR DESCRIPTION
**Pull request by anandjakhaniya**
**Jira Ticket:** https://vertaai.atlassian.net/browse/VR-10233

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java [hunk 1]
```diff
@@ -174,7 +253,19 @@ public void deleteExperimentRunComment(
...
               experimentRunEntity, request.getEntityId(), request.getId(), userInfo);
-      responseObserver.onNext(DeleteComment.Response.newBuilder().setStatus(status).build());
+      DeleteComment.Response response =
+          DeleteComment.Response.newBuilder().setStatus(status).build();
+      String projectId = experimentRunDAO.getProjectIdByExperimentRunId(request.getEntityId());
+      GetResourcesResponseItem responseItem =
+          roleService.getEntityResource(projectId, ModelDBServiceResourceTypes.PROJECT);
+      saveAuditLog(
+          Optional.of(userInfo),
+          ModelDBServiceActions.DELETE,
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java]: Update. We're updating the ER, not deleting it.

File: backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java [hunk 2]
```diff
@@ -106,7 +147,20 @@ public void addExperimentRunComment(
...
-      responseObserver.onNext(AddComment.Response.newBuilder().setComment(newComment).build());
+      AddComment.Response response =
+          AddComment.Response.newBuilder().setComment(newComment).build();
+
+      String projectId = experimentRunDAO.getProjectIdByExperimentRunId(request.getEntityId());
+      GetResourcesResponseItem responseItem =
+          roleService.getEntityResource(projectId, ModelDBServiceResourceTypes.PROJECT);
+      saveAuditLog(
+          Optional.of(userInfo),
+          ModelDBServiceActions.CREATE,
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java]: Same

**Review by conradoverta — APPROVED** [review]
Two asks for actions. LGTM otherwise

File: backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java]: Resolved

File: backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java (hunk 2 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java]: Resolved

**Pull request merged**